app/controllers/application_controller.rb

```diff
@@ -1,5 +1,27 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :mailbox, :conversation
+
+  def current_user
+    User.find_by(id: session[:user_id])
```

Comment:
This will always query, even if session[:user_id] is null. It will also query (but probably hit cache) each time it's executed. You can get round that by doing something like
```@user ||= User.find_by.... if session[:user_id]```